zingocli/src/lib.rs

```diff
@@ -299,14 +297,13 @@ impl ConfigTemplate {
         } else {
             None
         };
-        let seed = matches.value_of("seed").map(|s| s.to_string());
-        let viewing_key = matches.value_of("viewing-key").map(|s| s.to_string());
+        let from = matches.value_of("from").map(|s| s.to_string());
         let maybe_birthday = matches.value_of("birthday");
-        if seed.is_some() && maybe_birthday.is_none() {
+        if from.is_some() && maybe_birthday.is_none() {
             eprintln!("ERROR!");
             eprintln!(
-            "Please specify the wallet birthday (eg. '--birthday 600000') to restore from seed."
-        );
+                "Please specify the wallet birthday (eg. '--birthday 600000') to restore a wallet. (If you want to load the entire blockchain instead, you can use birthday 0. /this would require extensive time and computational resources)"
```

Comment:
```suggestion
                "Please specify the wallet birthday (eg. '--birthday 600000') to restore a wallet. (If you want to load the entire blockchain instead, you can use birthday 0. This would require extensive time and computational resources)"
```